test/workspace.js

```diff
@@ -249,4 +249,61 @@ describe('Workspace', function() {
       });
     });
   });
+
+  describe('Creating 2 projects from template', function() {
+    beforeEach(givenEmptySandbox);
+    afterEach(function setWorkspaceToSandboxDir() {
+      process.env.WORKSPACE_DIR = SANDBOX;
+    });
+
+    it('Create note app & hello world app in same directory and switching workspace',
+      function(done) {
+        async.series([
```

Comment:
You could clean this up by taking out all the inline functions and extracting some to the bottom of the file and some to `support.js`. Once that's done, you can write:

```
async.series([
  createSandboxDir.bind('/noteapp'),
  loadWorkspace.bind('/noteapp'),
  testFilesDoNotExist.bind('/noteapp'),
  createSandboxDir.bind('/helloworldapp'),
  ...
]);
```